### ISSUE TITLE
md-select doesnt work unless it has ng directive in it

### ISSUE THREAD

**Issue by Anoop-Goudar** [angular/material2#8406]
#### Bug, feature request, or proposal:
<md-select  placeholder="Message Type"
  [(ngModel)]="messageType" (ngModelChange)="getMessagesOfType(messageType)">
    <md-option [value]="Text">
      Text
    </md-option>
    <md-option [value]="Media">
      Media
    </md-option>
  </md-select>
This is not working unless I put *ngIf="true" directive inside md-select 

#### What is the expected behavior?
It should have worked without any dependency over *ngIf directive

#### What is the current behavior?
Working only after I put *ngIf directive

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
@angular/cli": "^1.5.0, @angular/material": "^2.0.0-beta.6,angular2-materialize": "^15.0.1, Ubuntu, typescript": "^2.6.1, Chrome

**Comment by crisbeto** [comment]
Can you post an example of the issue? [You can use this Plunkr as a base.](https://goo.gl/uDmqyY)

**Comment by Anoop-Goudar** [comment]
can you please give me access to that plunk ?
Another thing I noticed it md-options expects values to be given in array using *ngFor  , else options wont get selected. 
There is some connectivity with *ng directives for me to make md-select work properly.

**Comment by crisbeto** [comment]
You can press the "Fork" button to create a copy of the Plunk.

**Comment by Anoop-Goudar** [comment]
Actually I am using MdSelectModule not MatSelectModule. And was not able to import in the base plunk you sent

**Comment by jelbourn** [comment]
Closing this since it's referring to beta.6 which is now about 6 months old. Feel free to open a new issue if you're still able to reproduce with the latest version.
